sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MethodsShouldUseBaseTypes.cs

```diff
@@ -114,7 +114,7 @@ private static bool IsTrackedParameter(IParameterSymbol parameterSymbol)
                    !type.Is(KnownType.System_String);
         }
 
-        private SyntaxNode GetNextUnparenthesizedParent(SyntaxNode node)
+        private static SyntaxNode GetNextUnparenthesizedParent(SyntaxNode node)

```

Comment:
I think we have an helper method doing the exact same thing.

Reply:
Not exactly ;) This one is returning the parent of the top parenthesized expression...